shard_static/models.py

```diff
@@ -1,23 +1,23 @@
 from django.db import models
 
-from shard_static.managers import ShardStaticManager
+from shard_static.constants import ALL_SHARD_GROUP
```

Comment:
I think `ALL_SHARD_GROUP` is not proper.

I can not image how to add custom group and handle it by user.

Reply:
`ALL_SHARD_GROUP` means just that it's able to spread to all of shards.
The way for adding custom group is same the way for adding group to shard.
And it's already wrote docs.

Reply:
Can I set two shard_gourp on model? 
The code below will only work if it is ALL_SHARD_GROUP.
I am confused aboutn `allow_relation`. 
In short, `allow_relation` means can be FK.  [django](https://docs.djangoproject.com/en/2.0/topics/db/multi-db/#allow_relation)
What is the relationship `allow_relation` with ALL_SHARD_GROUP?


```
if isinstance(obj1, (ShardMixin, ShardStaticMixin)) and isinstance(obj2, (ShardMixin, ShardStaticMixin)):
             return obj1.shard_group == obj2.shard_group or obj1.shard_group == ALL_SHARD_GROUP or obj2.shard_group == ALL_SHARD_GROUP	             return obj1.shard_group == obj2.shard_group or obj1.shard_group == ALL_SHARD_GROUP or obj2.shard_group == ALL_SHARD_GROUP
```